src/main/scala/org/scalameter/reporting/HtmlReporter.scala

```diff
@@ -22,7 +22,8 @@ case class HtmlReporter[T: Numeric](embedDsv: Boolean = true) extends Reporter[T
       new DsvReporter(dsvDelimiter).report(results, persistor)
     }
 
-    val resultdir = results.context(reports.resultDir)
+    // resultDir is global setting
+    val resultdir = currentContext(reports.resultDir)
     val root = new File(resultdir, "report")
```

Comment:
That sounds like a good idea. It might be better if this was treated as a global setting, which is updated only once when the command line is parsed.